.github/workflows/ui-rm_head_upgrade.yaml

```diff
@@ -33,8 +33,19 @@ on:
         default: v1.28.8+k3s1
         type: string
         required: true
+      k8s_version_upgrade_ds_cluster:
+        description: Proceed with K8s version Upgrade (True/False)
+        default: false
+        required: false
+        type: boolean
+      k8s_version_to_upgrade_ds_cluster_to:
+        description: K3s version to be upgraded on cluster. Provide version when k8s upgrade option is checked.
+        default: 'v1.31.8+k3s1'
+        required: false
+        type: string
```

Comment:
why not to add `boolean` here as previously?

Reply:
No `boolean` required as we have to take it string from the user.